### DISCUSSION TITLE
New FBA Inbound API - Not providing EXP on SKU that requires EXP is NOT producing error -- only empty results

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4353] [General]
We have a SKU that is a food item (expiration required)

If we do not provide "expiration" during createInboundPlan, there is NO error, however when we get to "listPackingGroups", it returns with an empty array.

There seems to be no clean way to tell when the error is due to not providing an expiration date when one is required.

As a secondary issue, it seems required that we provide an expiration date up front -- but this is not reasonable without pulling and packing the items.  So just to even find out if items can be packed together (ie: if they require multiple packing groups), we need to go to shelves and find expiration dates at the beginning of the process.

Guidance?

**Comment by puppsupr** [comment]
HI [MetaMan2023](https://github.com/MetaMan2023),
In order to correctly resolve this issue, Please [open a support case](https://developer.amazonservices.com/support) following the [guidance](https://developer.amazonservices.com/support-guidelines) here so we can pursue the investigation as additional details are required from your end.

**Comment by MetaMan2023** [comment]
I believe the issue has been resolved, but will follow that path if so.  Thank you.
